alembic/autogenerate/render.py

```diff
@@ -409,8 +409,9 @@ def _drop_constraint(
         "prefix": _alembic_autogenerate_prefix(autogen_context),
         "name": _render_gen_name(autogen_context, op.constraint_name),
         "table_name": _ident(op.table_name),
-        "type": (", type_=%r" %
-                 _ident(op.constraint_type)) if op.constraint_type else "",
+        "type": (", type_=%r" % _ident(op.constraint_type))
```

Comment:
may make sense to use f-strings here

Reply:
That would get my vote.  I didn't want to rock the boat too much though so was just sticking with the existing convention in this file.

Reply:
The current behaviour is mere or less to update string formatting if a part of file gets updated for other reasons, so for the new things frel free to use f-strings

Reply:
Understood.  I've just pushed an additional commit to convert this particular function to an f-string based implementation.